src/views/ThreeWActivityDetail/ActivityListItem/index.tsx

```diff
@@ -0,0 +1,153 @@
+import { useMemo } from 'react';
+import { isDefined, listToMap } from '@togglecorp/fujs';
+
+import TextOutput from '#components/TextOutput';
+import Container from '#components/Container';
+import useTranslation from '#hooks/useTranslation';
+import { useRequest } from '#utils/restRequest';
+
+import i18n from './i18n.json';
+import styles from './styles.module.css';
+
+export type Props = {
+    maleCount: number | null | undefined;
+    femaleCount: number | null | undefined;
+    peopleCount: number | null | undefined;
+    householdCount: number | null | undefined;
+    title: string | null | undefined;
+    customAction: string | null | undefined;
+    sectorDetails: string | null | undefined;
+    peopleHouseholds: 'people' | 'households' | '' | null | undefined;
+    customSupply: {
+        [key: string]: number;
+    };
+    activitySupply: {
+        [key: string]: number;
+    } | undefined;
+    activityDetails: string | null | undefined;
+    isSimplifiedReport: boolean | null | undefined;
+}
+
+function ActivityListItem(props: Props) {
+    const {
+        maleCount,
+        femaleCount,
+        peopleCount,
+        householdCount,
+        peopleHouseholds,
+        title,
+        sectorDetails,
+        customAction,
+        customSupply,
+        activitySupply,
+        activityDetails,
+        isSimplifiedReport,
+    } = props;
+    const strings = useTranslation(i18n);
+
+    const {
+        response: optionsResponse,
+    } = useRequest({
+        url: '/api/v2/emergency-project/options/',
+    });
```

Comment:
We should not call request on an item that is being rendered on a list.
This will call the same request for each item.